packages/victory-core/src/victory-state/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./victory-provider";
 export * from "./types";
+export * from "./victory-provider-maybe";
```

Comment:
No, it's kinda a weird name, but the idea is that sometimes a Victory component is inside a container, sometimes it's standalone.  This `VictoryProviderMaybe` HOC will detect if there's already a parent, and use it, but if it's missing, it'll add it.  So that's why I called it "maybe".